[2024-09-18 00:13:29,871: INFO: main: Hello world]
[2024-09-18 00:15:36,290: INFO: main: Hello world]
[2024-09-18 01:42:57,902: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-09-18 01:42:57,915: INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-18 01:42:57,921: INFO: common: created directory at: artifacts]
[2024-09-18 01:42:57,924: INFO: common: created directory at: artifacts/data_ingestion]
[2024-09-18 01:42:57,927: INFO: 986331348: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-09-18 01:43:10,278: INFO: 986331348: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-09-18 02:10:37,160: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-09-18 02:10:37,165: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-09-18 02:10:37,169: INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-18 02:10:37,175: INFO: common: created directory at: artifacts]
[2024-09-18 02:10:37,185: INFO: common: created directory at: artifacts/data_ingestion]
[2024-09-18 02:10:37,189: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-09-18 02:10:49,687: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-09-18 02:10:51,414: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-09-18 14:01:03,637: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-09-18 14:01:03,644: INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-18 14:01:03,648: INFO: common: created directory at: artifacts]
[2024-09-18 14:01:03,650: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-09-18 14:01:23,052: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-09-18 16:38:02,535: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-09-18 16:38:02,541: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-09-18 16:38:02,547: INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-18 16:38:02,548: INFO: common: created directory at: artifacts]
[2024-09-18 16:38:02,549: INFO: common: created directory at: artifacts/data_ingestion]
[2024-09-18 16:38:02,550: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-09-18 16:38:18,770: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-09-18 16:38:20,048: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-09-18 16:38:20,048: INFO: main: >>>>>> stage Prepare Base Model started <<<<<<]
[2024-09-18 16:38:20,056: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-09-18 16:38:20,060: INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-18 16:38:20,061: INFO: common: created directory at: artifacts]
[2024-09-18 16:38:20,062: ERROR: main: 'ConfigurationManager' object has no attribute 'get_prepare_base_model_config']
Traceback (most recent call last):
  File "main.py", line 19, in <module>
    data_ingestion.main()
  File "c:\users\vishv\onedrive\desktop\vit\kidney-disease-classification\src\cnnClassifier\pipeline\stage_02_prepare_base_model.py", line 13, in main
    prepare_base_model_config = config.get_prepare_base_model_config()
AttributeError: 'ConfigurationManager' object has no attribute 'get_prepare_base_model_config'
[2024-09-18 16:40:03,662: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-09-18 16:40:03,667: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-09-18 16:40:03,673: INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-18 16:40:03,674: INFO: common: created directory at: artifacts]
[2024-09-18 16:40:03,674: ERROR: main: 'ConfigurationManager' object has no attribute 'get_data_ingestion_config']
Traceback (most recent call last):
  File "main.py", line 9, in <module>
    data_ingestion.main()
  File "c:\users\vishv\onedrive\desktop\vit\kidney-disease-classification\src\cnnClassifier\pipeline\stage_01_data_ingestion.py", line 13, in main
    data_ingestion_config = config.get_data_ingestion_config()
AttributeError: 'ConfigurationManager' object has no attribute 'get_data_ingestion_config'
[2024-09-18 16:42:18,537: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-09-18 16:42:18,543: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-09-18 16:42:18,547: INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-18 16:42:18,549: INFO: common: created directory at: artifacts]
[2024-09-18 16:42:18,549: ERROR: main: 'ConfigurationManager' object has no attribute 'get_data_ingestion_config']
Traceback (most recent call last):
  File "main.py", line 9, in <module>
    data_ingestion.main()
  File "c:\users\vishv\onedrive\desktop\vit\kidney-disease-classification\src\cnnClassifier\pipeline\stage_01_data_ingestion.py", line 13, in main
    data_ingestion_config = config.get_data_ingestion_config()
AttributeError: 'ConfigurationManager' object has no attribute 'get_data_ingestion_config'
[2024-09-18 16:43:03,796: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-09-18 16:43:03,803: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-09-18 16:43:03,810: INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-18 16:43:03,811: INFO: common: created directory at: artifacts]
[2024-09-18 16:43:03,811: ERROR: main: 'ConfigurationManager' object has no attribute 'get_data_ingestion_config']
Traceback (most recent call last):
  File "main.py", line 9, in <module>
    data_ingestion.main()
  File "c:\users\vishv\onedrive\desktop\vit\kidney-disease-classification\src\cnnClassifier\pipeline\stage_01_data_ingestion.py", line 13, in main
    data_ingestion_config = config.get_data_ingestion_config()
AttributeError: 'ConfigurationManager' object has no attribute 'get_data_ingestion_config'
[2024-09-18 16:44:47,222: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-09-18 16:44:47,228: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-09-18 16:44:47,233: INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-18 16:44:47,234: INFO: common: created directory at: artifacts]
[2024-09-18 16:44:47,237: ERROR: main: 'ConfigurationManager' object has no attribute 'get_data_ingestion_config']
Traceback (most recent call last):
  File "main.py", line 9, in <module>
    data_ingestion.main()
  File "c:\users\vishv\onedrive\desktop\vit\kidney-disease-classification\src\cnnClassifier\pipeline\stage_01_data_ingestion.py", line 13, in main
    data_ingestion_config = config.get_data_ingestion_config()
AttributeError: 'ConfigurationManager' object has no attribute 'get_data_ingestion_config'
[2024-09-18 18:02:51,469: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-09-18 18:02:51,477: INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-18 18:02:51,480: INFO: common: created directory at: artifacts]
[2024-09-18 18:02:51,483: INFO: common: created directory at: artifacts/data_ingestion]
[2024-09-18 18:02:51,486: INFO: 986331348: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-09-18 18:03:07,484: INFO: 986331348: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-09-18 18:07:30,972: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-09-18 18:07:30,978: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-09-18 18:07:30,982: INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-18 18:07:30,983: INFO: common: created directory at: artifacts]
[2024-09-18 18:07:30,983: ERROR: main: 'ConfigurationManager' object has no attribute 'get_data_ingestion_config']
Traceback (most recent call last):
  File "main.py", line 9, in <module>
    data_ingestion.main()
  File "c:\users\vishv\onedrive\desktop\vit\kidney-disease-classification\src\cnnClassifier\pipeline\stage_01_data_ingestion.py", line 13, in main
    data_ingestion_config = config.get_data_ingestion_config()
AttributeError: 'ConfigurationManager' object has no attribute 'get_data_ingestion_config'
[2024-09-18 18:09:31,565: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-09-18 18:09:31,570: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-09-18 18:09:31,575: INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-18 18:09:31,576: INFO: common: created directory at: artifacts]
[2024-09-18 18:09:31,577: INFO: common: created directory at: artifacts/data_ingestion]
[2024-09-18 18:09:31,578: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-09-18 18:09:50,994: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-09-18 18:09:52,210: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-09-18 18:09:52,210: INFO: main: >>>>>> stage Prepare Base Model started <<<<<<]
[2024-09-18 18:09:52,217: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-09-18 18:09:52,221: INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-18 18:09:52,222: INFO: common: created directory at: artifacts]
[2024-09-18 18:09:52,223: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-09-18 18:09:53,531: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-09-18 18:09:53,972: INFO: main: >>>>>> stage Prepare Base Model completed <<<<<<

x==========x]
